src/overlays/effects/ovl_Effect_Ss_En_Ice/z_eff_ss_en_ice.c

```diff
@@ -1,30 +1,157 @@
 /*
  * File: z_eff_ss_en_ice.c
  * Overlay: ovl_Effect_Ss_En_Ice
- * Description:
+ * Description: Ice clumps
```

Comment:
Maybe not in this PR, but after all the effects are done, it'd be good to possibly have a summary of all the ice and frozen textures from effects/gameplay keep. I feel like there's a lot of them... It'd be good to know how they're all differentiated from each other from bigger picture

Reply:
Yeah, there are a lot of assets in gameplay keep from effects that could use a good cleaning. Mostly they are just distinguished with numbers, both here and in OoT.